App Delegate/MIT_MobileAppDelegate.m

```diff
@@ -699,6 +701,22 @@ - (void)saveModulesState {
     [[NSUserDefaults standardUserDefaults] setObject:nil forKey:MITModulesSavedStateKey];
 }
 
+- (void)setLastActiveModuleName:(NSString *)lastActiveModuleName
```

Comment:
Line 773 in _-[MIT_MobileAppDelegate slidingViewController:didShowTopViewController:]_